platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/MerkleDbDataSource.java

```diff
@@ -316,14 +321,14 @@ public MerkleDbDataSource(
         // key to path store
         String keyToPathStoreName = tableName + "_objectkeytopath";
         keyToPath = new HalfDiskHashMap(
-                database.getConfig(),
+                database.getConfiguration(),
```

Comment:
`HalfDiskHashMap`, similar to `MemoryIndexDiskKeyValueStore`, only needs a `MerkleDbConfig`, not the whole `Configuration` object